compute/admin_guide/configure/configure-agentless-scanning.adoc

```diff
@@ -50,73 +31,74 @@ ifdef::prisma_cloud[]
 
 endif::prisma_cloud[]
 
-[WARNING]
-====
-When no custom security group is provided, the Prisma Cloud Console uses the default VPC security group for AWS and GCP accounts to connect.
-If the default security groups are not available the connection fails. 
-You must create custom security groups in AWS and GCP manually and specify their name or identifier when you configure Agentless Scanning.
-====
-
 [#_individual-account]
 [.task]
-=== Configure an Individual Account for Scanning
+=== Configure Agentless Scanning
 
-When configuring a single account for agentless scanning, you have all the configuration fields available for that account's provider.
-<<_configuration-fields,These fields change from one cloud provider to another>>, and even accounts from the same provider might require different configuration values depending on your use case.
-To provide you the most flexibility, you can setup configuration scanning when you add the cloud provider account or change the configuration later.
-
-The following procedure shows the steps needed to configure agentless scanning.
+The following procedure shows the steps needed to configure agentless scanning for a cloud account.
 
 [.procedure]
-. Go to *Manage > Cloud accounts* 
+. Go to *Manage > Cloud accounts* inside Compute
 +
 image::manage-cloud-accounts.png[width=800]
 
 . Click on *Add Account* or click the edit icon for an existing account.
 +
 image::add-account.png[width=400]
 
-. Select your cloud provider, configure its https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/authentication/credentials_store[credentials], and click *Next*.
+. Select your cloud provider, add its https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/authentication/credentials_store[credential].
 +
 image::set-credentials.png[width=800]
++
+NOTE: You can leave permissions empty while creating these credentials. Prisma Cloud will provide Agentless permission templates in the next step after validating the given credentials. For incorrect credentials, permissions template download will show an error.
```

Comment:
@Aqsaxvv092 The behavior in this note isn't clear. What fields can be left empty? Do you mean the fields listed in lines 55-57 for each provider?

Reply:
@Aqsaxvv092 we are merging this PR now, but we need an answer from you to resolve this question.

Reply:
Users who create credentials in public clouds like AWS. This is something they do on cloud side not on our side. While creating credentials in AWS there is a step in which they can provide permissions to the credential / token. They need to just leave it empty during that stage.